fe/fe-core/src/main/java/org/apache/doris/datasource/property/PropertyConverter.java

```diff
@@ -267,6 +267,10 @@ private static void setS3FsAccess(Map<String, String> s3Properties, Map<String,
         String credentialsProviders = s3Properties
                 .getOrDefault(S3Properties.CREDENTIALS_PROVIDER, defaultProviderList);
         s3Properties.put(Constants.AWS_CREDENTIALS_PROVIDER, credentialsProviders);
+        if (properties.get(S3Properties.Env.ENDPOINT) != null
+                && properties.get(S3Properties.Env.ENDPOINT).toLowerCase(Locale.ROOT).startsWith("obs")) {
+            s3Properties.put("fs.s3a.connection.ssl.enabled", "false");
```

Comment:
you should use obs.endpoint(see ObsProperties.ENDPOINT)，AWS_ENDPOINT is just reserved for compatibility.
and AWS_ENDPOINT is s3 properties，you must not use it.

Reply:
OK, The previous is compatible with s3 modification, and then directly used OBS properties
    "fs.obs.access.key" = "xx",
    "fs.obs.secret.key" = "xx",
    "fs.obs.endpoint" = "xx"